Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -202,7 +204,8 @@ internal object[] GetValues(BindableProperty property0, BindableProperty propert
 			return values;
 		}
 
-		internal object[] GetValues(BindableProperty property0, BindableProperty property1, BindableProperty property2)
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public object[] GetValues(BindableProperty property0, BindableProperty property1, BindableProperty property2)
 		{
```

Comment:
I'd guess the `params` keyword was not used here in order to prevent GC allocation of the array; that signature could be an optimization. For this PR, I'd prefer not to make any control flow or signature modifications to ensure there are no regressions. More to the point, this API was not requested to be put on a controller iface. So this change is not expanding our supported surface area.